cmd/world/forge/deployment.go

```
@@ -128,3 +133,130 @@
+	result, err := sendRequest(ctx, http.MethodGet, statusURL, nil)
+	if err != nil {
+		return eris.Wrap(err, "Failed to get deployment status")
+	}
+	var response map[string]any
+	err = json.Unmarshal(result, &response)
+	if err != nil {
+		return eris.Wrap(err, "Failed to unmarshal deployment status")
+	}
+	var data map[string]any
+	if response["data"] != nil {
+		data = response["data"].(map[string]any)
```

golangci-lint (1.62.2)

Error return value is not checked

(errcheck)